azure-devops/azext_devops/dev/pipelines/pipeline_create.py

```diff
@@ -241,13 +252,22 @@ def _get_repo_name_from_repo_url(repository_url):
             if stripped_path.endswith('.git'):
                 stripped_path = stripped_path[:-4]
             return stripped_path
-    if repo_type == _AZURE_GIT_REPO_TYPE:
+    if repo_type.lower() == _AZURE_GIT_REPO_TYPE:
```

Comment:
Will `lower()` work `repo_type`?, `_AZURE_GIT_REPO_TYPE` in line 45 is defined as `TfsGit`

Reply:
good observation